Snowflake.Data/Core/FileTransfer/StorageClient/ISFRemoteStorageClient.cs

```diff
@@ -11,29 +11,29 @@ namespace Snowflake.Data.Core.FileTransfer
 {
     internal class WrappedContentInfo
     {
-        public string KeyId;
-        public string EncryptedKey;
-        public string Algorithm;
+        public string KeyId { get; set; }
+        public string EncryptedKey { get; set; }
+        public string Algorithm { get; set; }
```

Comment:
Do we really need it? This file has 100% code coverage regardless the change.
If it is only for increasing numerator and denominator of the code coverage factor, I 'd remove it.

Reply:
It's not really needed since it was only done for code coverage numbers. I'll remove these changes then